### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
## UI/ pages
- profile page
- bookmark page
- notification page

## comp
- update: navbar comp
- update Profile comp

### CODE REVIEW

File: src/components/editProfileModel/editProfileModel.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import Modal from "@mui/material/Modal";
+import { TextField } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import IconButton from "@mui/material/IconButton";
+import PhotoCamera from "@mui/icons-material/PhotoCamera";
```
Comment (Chiragmodi01) [inline src/components/editProfileModel/editProfileModel.js]: Using barrel imports for `mui` will be helpful, there'll be a lo of them in almost every comp

**Review by Chiragmodi01 — COMMENTED** [review]
Nothing much to comment on code, as it's pretty concise.
UI looks dull to me as of now, maybe experiment with mild shadow on boxes and font-weight.

File: src/App.jsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ function App() {
         <Route path="/Signup" element={<Signup />} />
         <Route path="/profile" element={<Profile />} />
         <Route path="/bookmark" element={<Bookmark />} />
+        <Route path="/notification" element={<Notification />} />
       </Routes>
       <RightSideBar />
```
Comment (agrawalshanu708) [inline src/App.jsx]: use navlink

**Review by agrawalshanu708 — APPROVED** [review]
work on UI Part
codes looks clean

File: src/components/editProfileModel/editProfileModel.js (hunk 1 above)
Comment (harshitpaliwal95) [reply src/components/editProfileModel/editProfileModel.js]: i'll try to change it in next pr

**Pull request merged**